packages/docusaurus-plugin-content-docs/src/slug.ts

```diff
@@ -18,6 +18,7 @@ import {
 import type {DocMetadataBase} from './types';
 import {isConventionalDocIndex} from './docs';
 import type {NumberPrefixParser} from '@docusaurus/plugin-content-docs';
+import path from 'path';
 
 export default function getSlug({
```

Comment:
here we won't have acces to the generator `isCategoryIndex`

IMHO it's probably not a big deal because we have the `slug` frontmatter anyway 🤷‍♂️ should be fine

Reply:
Yeah, I don't plan to expose this to the slug function, because category index pages are not coupled to the slug system and we'd better not give users the impression that they are😅 Had I started by applying the logic to sluggification, I would have made `isCategoryIndex` a root option instead of a prop to sidebar generator.